src/pop_verifier_old_code

```diff
@@ -0,0 +1,181 @@
+%% @doc code that handles running a full verifier node.
```

Comment:
just delete and put faith in version control 😱

Reply:
:scream: